client/my-sites/media-library/content.jsx

```diff
@@ -135,6 +135,9 @@ class MediaLibraryContent extends React.Component {
 						i18nOptions
 					);
 					break;
+				case MediaValidationErrors.SERVICE_AUTH_FAILED:
+					return this.getPicasaUpgradeNotice( errorType, onDismiss );
```

Comment:
We don't return from the other switch cases but set some variable to configure the notice. I find it less error-prone when we follow the same mechanism consistently. Can we change this so every case behaves the same?